src/models/data.py

```diff
@@ -516,6 +536,7 @@ def calculate_normalizing_array(
         return normalizing_array
 
     def _calculate_aggs(self, x: xr.Dataset) -> np.ndarray:
+        warnings.warn("Deprecated for causing the static data to vary")
```

Comment:
I think i want to remove this function right? Stops the static data from being static ..?

Reply:
Hmm... I see an argument for keeping this. It stops certain dimensions in the static data from being static, but then we can just do `calculate_aggs=False` in the model. On the other hand, I do think instance-wise aggregations could be useful

Reply:
Okay they could just go in the dynamic data instead then?